### DISCUSSION TITLE
Realeses

### DISCUSSION THREAD

**Discussion by Sxars22ww** [azahar-emu/azahar#549] [Q&A] [answered]
Did they delete all the Realeses it's not showing any for me

**Comment by rtiangha** [comment]
You can still find a copy of Lime3DS on Google Play.

You may also get lucky with the Wayback Machine:
https://web.archive.org/web/20241115114434/github.com/lime3ds/lime3ds/releases

**Comment by OpenSauce04** [comment, marked as answer]
https://github.com/Lime3DS/lime3ds-archive/releases
